app/models/pydantic/assets.py

```diff
@@ -25,9 +25,14 @@ class AssetCreateIn(BaseModel):
     asset_type: AssetType
     asset_uri: Optional[str]
     is_managed: bool = True
-    creation_options: CreationOptions
+    creation_options: OtherCreationOptions
+    fields_value: Optional[List[Dict[str, Any]]]
     metadata: Optional[AssetMetadata]
 
+    class Config:
+        extra = "forbid"
```

Comment:
What does forbid do? Aren't any fields which are not specified simply ignored?

Reply:
https://pydantic-docs.helpmanual.io/usage/model_config/
"forbid" means that extra fields in the request cause an error. Originally I did this because it was suggested that it helps Pydantic choose which model to validate against. I'm no longer sure it is strictly necessary, I can remove it and see if requests are still categorized correctly.
However I think disallowing extra fields has the benefit of providing feedback if someone is trying to use the API incorrectly, rather than silently ignoring what might be typos, etc. What do you think?

Reply:
ok, sounds good. It seems like this way you disovered the `fields` and `zipped` attributes which you added back in, right?
I think they were both remainders from previous implementations which I didn't clean up properly.
So rather than including them back into the model, they should be fully removed from the code

Reply:
Got it. Yeah, satisfying Pydantic with extra=forbid has been a pain, but it has made me find lots of cases of the wrong models being used that would have been a lot harder to find otherwise.